LICENSE.md

```diff
@@ -0,0 +1,26 @@
+BSD 3-Clause License
+
```

Comment:
You might consider a copyright notice here as shown: https://opensource.org/licenses/BSD-3-Clause